react/features/filmstrip/components/native/Filmstrip.js

```diff
@@ -110,35 +160,16 @@ class Filmstrip extends Component<Props> {
                         && !isNarrowAspectRatio_
                         && <LocalThumbnail />
                 }
-                <ScrollView
+                <FlatList
+                    data = { this._getFilmstripData() }
                     horizontal = { isNarrowAspectRatio_ }
+                    onViewableItemsChanged = { this._onViewableItemsChanged }
+                    removeClippedSubviews = { true }
+                    renderItem = { this._renderItem }
                     showsHorizontalScrollIndicator = { false }
                     showsVerticalScrollIndicator = { false }
-                    style = { styles.scrollView } >
-                    {
-                        !this._separateLocalThumbnail
-                            && !isNarrowAspectRatio_
-                            && <LocalThumbnail />
-                    }
-                    {
-                        /* eslint-disable react/jsx-wrap-multilines */
-
-                        this._sort(
-                                this.props._participants,
-                                isNarrowAspectRatio_)
-                            .map(p =>
-                                <Thumbnail
-                                    key = { p.id }
-                                    participant = { p } />)
-
-                        /* eslint-enable react/jsx-wrap-multilines */
-                    }
-                    {
-                        !this._separateLocalThumbnail
-                            && isNarrowAspectRatio_
-                            && <LocalThumbnail />
-                    }
-                </ScrollView>
+                    style = { styles.scrollView }
```

Comment:
can we rename this style? maybe to something a bit more generic and not tied to the actual component holding the thumbnails?